opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -72,7 +72,7 @@
 )
 from opentelemetry.semconv.resource import ResourceAttributes
 
-LabelValue = typing.Union[str, bool, int, float]
+LabelValue = typing.Union[str, bool, int, float, list[str]]
```

Comment:
While at it can you make these alias for the types defined here? https://github.com/open-telemetry/opentelemetry-python/blob/2098c5644588cd4e766eb8e31a1ca26c4e00ea46/opentelemetry-api/src/opentelemetry/util/types.py#L18-L44 
It will remain backward compatible and accurate.